<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibrate_import_export</class>
 <widget class="QWidget" name="calibrate_import_export">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="restore_button">
     <property name="text">
      <string>Restore Factory Calibration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vlayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="export_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="import_button">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="text_edit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
